src/services/ParticipantProjectService.ts

```diff
@@ -168,6 +169,7 @@ const list = async (
 };
 
 const update = async (participantProject: ParticipantProject): Promise<Response> => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
there are no unused vars here, interesting

Reply:
projectId is not used. Will fix it